_layouts/api.html

```diff
@@ -9,6 +9,12 @@
         {% if page.pill == 'overview' %} class="active" {% endif %}>
         <a href="/documentation/api">Overview</a></li>
         <li
+        {% if page.pill == 'overview' %} class="active" {% endif %}>
+        <a href="/documentation/api#android">Android</a></li>
+        <li
+        {% if page.pill == 'overview' %} class="active" {% endif %}>
+        <a href="/documentation/api#spring">Android</a></li>
+        <li
```

Comment:
This changes the 1-1 mapping between navigation and pages and breaks the consistency of the convention used for the rest of the site.